### DISCUSSION TITLE
creating a tun

### DISCUSSION THREAD

**Discussion by khromenokroman** [strongswan/strongswan#1924] [Q&A] [answered]
Please tell me strongswan can create tun adapters?

**Comment by tobiasbrunner** [comment, marked as answer]
What for? In what context? Maybe look at [route-based VPNs](https://docs.strongswan.org/docs/5.9/features/routeBasedVpn.html) in the docs?

**Reply by khromenokroman** [reply]
Thank you very much this is what you need
